src/Modules/Legacy/Inverse/SolveInverseProblemWithTikhonov.h

```diff
@@ -61,7 +62,7 @@ namespace Inverse {
 		MODULE_TRAITS_AND_INFO(ModuleHasUIAndAlgorithm)
 
 	private:
-		// void update_lcurve_gui(const double lambda, const Core::Algorithms::Inverse::SolveInverseProblemWithTikhonovImpl_child::LCurveInput& input, const int lambda_index);
+		virtual void update_lcurve_gui(const SCIRun::Core::Datatypes::DenseMatrixHandle& lambda, const SCIRun::Core::Datatypes::DenseMatrixHandle& input, const SCIRun::Core::Datatypes::DenseMatrixHandle& lambda_index);
```

Comment:
Even better, the `update_lcurve_gui` functions should go in separate classes, then used by the modules.